Project1/CellPath.h

```diff
@@ -1,17 +1,19 @@
 #pragma once
-#include "GridCell.h"
+#include <list>
+#include "MobilityList.h"
+class GridCell;
 //TODO(Torrey): Create Superclasses of this. EX: UnitPath has a unit to traverse the Map.
 using CellWrap = std::reference_wrapper<GridCell>;
```

Comment:
Can we rename this CellRef? I think I saw somewhere else where you used that naming convention for a reference wrapper and I like it better.